src/common/registry/storage/operator.storage.ts

```diff
@@ -23,10 +23,6 @@ export class RegistryOperatorStorageService {
       .select(fields || '*')
       .from<RegistryOperator>('registry_operator')
       .where(where)
-      .orderBy([
```

Comment:
I don't want to remove that. Let's just keep it in case something goes wrong with knex/db.

Reply:
moduleAddress we don't need there
we usually have it in where

Reply:
reverted